fly.toml

```diff
@@ -0,0 +1,37 @@
+app = 'wagtail-news-template-{{ project_name }}'
+primary_region = 'lhr'
```

Comment:
If we don't provide this, will the `fly` CLI app ask for it? If so, maybe that's better. If not, we should document this so people make the right decision for their audience.

Reply:
It seems if you don't include it, Fly attempts to guess a region based on the "the fastest region for you". Oddly for me, this is Secaucus, NJ (US), which doesn't sound quite right. The prompt which says this however lets people tweak the setting, where they can then select it as they need to.

I suspect then we're going to need to do both - remove the default so a _potentially_ more sensible default is used, but also document that they may want to change it.